apps/frontend/src/locales/en-US/index.json

```diff
@@ -978,7 +978,7 @@
     "message": "Invitation to join project"
   },
   "project-member-header.invitation-with-role": {
-    "message": "You've been invited be a member of this project with the role of '{role}'."
+    "message": "You've been invited to be a member of this project with the role of ''{role}''."
```

Comment:
this seems to only be changed in the generated file and not in the source? so it'll be overridden next time intl:extract is run.

also what's with the double single-quotes?